src/luamin.js

```diff
@@ -1820,19 +1820,19 @@ function AddVariableInfo(ast) {
         }
         let self = currentScope
         currentScope.GetVar = function(varName){
-            self.VariableList.forEach((_var) => {
+             for(const _var of self.VariableList) {
```

Comment:
lots of .foreach that are supposed to return, but an inline foreach will not return